### ISSUE TITLE
React 18 support?

### ISSUE THREAD

**Issue by adityapatadia** [readmeio/markdown#851]
We are not able to add this in Next.js 14. Can you add support for react 18?

**Comment by rafegoldberg** [comment]
<img align="right" height="100" src="https://user-images.githubusercontent.com/886627/87183626-529d9580-c29b-11ea-8c54-5ca58a1f4a77.png" />

## This issue was resolved!

🚀 Changes included in [<b>v6.78.0</b>](/readmeio/markdown/releases/tag/v6.78.0)
<br>

**Comment by kellyjosephprice** [comment]
This should support react 18 now!

There is a warning about `react-codemirror2` not supporting react 8, but it seems fine to me.
```
$ npm ls react
npm ERR! code ELSPROBLEMS
npm ERR! invalid: react@18.2.0 /home/kelly/wsp/rdmd-react-18/node_modules/react
rdmd-react-18@0.1.0 /home/kelly/wsp/rdmd-react-18
├─┬ @readme/markdown@6.78.0
│ ├─┬ @readme/syntax-highlighter@13.0.0
│ │ ├─┬ react-codemirror2@7.3.0
│ │ │ └── react@18.2.0 deduped invalid: ">=15.5 <=17.x" from node_modules/@readme/syntax-highlighter/node_modules/react-codemirror2
│ │ └── react@18.2.0 deduped
│ ├─┬ @readme/variable@16.0.0
│ │ └── react@18.2.0 deduped
│ ├─┬ @tippyjs/react@4.2.6
│ │ └── react@18.2.0 deduped
│ └── react@18.2.0 deduped
├─┬ @testing-library/react@13.4.0
│ └── react@18.2.0 deduped
├─┬ react-dom@18.2.0
│ └── react@18.2.0 deduped
├─┬ react-scripts@5.0.1
│ └── react@18.2.0 deduped
└── react@18.2.0
```
